cpu/cpupower.py

```diff
@@ -32,8 +31,9 @@ class cpupower(Test):
     """
 
     def setUp(self):
-        if not os.path.exists('/sys/devices/system/cpu/cpu0/cpufreq'):
-            self.cancel('sysfs directory for cpufreq is unavailable.')
+        if not 'PowerNV' in cpu._get_cpu_info():
```

Comment:
Suggestion: Better to add a similar implementation of https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/ras/ras.py#L58

Reply:
i think should be ok as in none of any combination on this test should run on any other arch